packages/experimental/RadioGroup/src/Radio/Radio.types.ts

```diff
@@ -8,13 +8,28 @@ import { Variant } from '@fluentui-react-native/framework';
 export const radioName = 'Radio';
 
 export interface RadioTokens extends FontTokens, IColorTokens, IForegroundColorTokens, IBackgroundColorTokens, IBorderTokens {
+  /**
+   * Total size of the Radio View
+   */
+  radioViewSize?: number;
+
+  /**
+   * Horizontal padding of the root view
+   */
+  rootHorizontalPadding?: number;
```

Comment:
What is the intent to create these tokens instead of just using in styles? Is  one way better than other?

Reply:
There were platform differences in the value, would it be better to just hardcode the values under a platform check?